### PR TITLE
Restrict GA message

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
As described on https://github.com/netdata/product/issues/340 , we are sending undesired messages to GA. This PR parses the tag message avoiding the send of undesired messages to GA.

##### Component Name
Report

### CODE REVIEW

File: libnetdata/log/log.c [hunk 1]
```diff
@@ -822,7 +822,8 @@ void fatal_int( const char *file, const char *function, const unsigned long line
     char action_data[70+1];
 	snprintfz(action_data, 70, "%04lu@%-10.10s:%-15.15s/%d", line, file, function, __errno);
 	char action_result[60+1];
-	snprintfz(action_result, 60, "%s:%s",program_name, strcmp(program_name,"STREAM_RECEIVER")?netdata_thread_tag():"[x]");
+
```
Comment (mfundul) [inline libnetdata/log/log.c]: This PR is a good opportunity to remove those tab characters here and convert them to spaces.

Comment (thiagoftsm) [reply libnetdata/log/log.c]: Done! Thank you observe this!

Comment (mfundul) [reply libnetdata/log/log.c]: The comparison got lost though.

Comment (mfundul) [reply libnetdata/log/log.c]: no, I'm wrong, disregard.

Comment (thiagoftsm) [reply libnetdata/log/log.c]: All right!

**Pull request merged**